framework/content/ecl-colors/README.md

```diff
@@ -0,0 +1,16 @@
+# Colors
```

Comment:
Please also add the documentation from the design wiki pages: https://webgate.ec.europa.eu/CITnet/confluence/pages/viewpage.action?pageId=650250290#Typography..-Fontcolour